### DISCUSSION TITLE
change color of text or background in console windows

### DISCUSSION THREAD

**Discussion by pnanu175** [community/community#29963] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to change color of text and background in console , how can I do it

**Comment by thanhngan22** [comment, marked as answer]
you can use some solutions: 

1.  system ( "color ab"); 
     a  ( background )
     b ( characters ).
2. add this func: 
    void set_color ( int code ) {
    HANDLE color = GetStdHandle(STD_OUTPUT_HANDLE);
    SetConsoleTextAttribute( color , code );
}

**Reply by pnanu175** [reply]
thank you for this helpful answer
